src/raft_server.c

```diff
@@ -183,17 +185,18 @@ raft_node_t* raft_add_node_internal(raft_server_t* me, raft_entry_t *ety, void*
     raft_node_t* node = raft_get_node(me, id);
     if (node)
     {
-        if (!raft_node_is_voting(node))
+        // node can be promoted only from a non voting state into a voting one
```

Comment:
```suggestion
        /* node can be promoted only from a non voting state into a voting one */
```

We have `/* */` for comments generally. (just to be consistent) (Also `//` comments are not C89, we shouldn't have it in the API by mistake if we want to maintain C89 API.) 

I see we introduced couple of `//`, we should fix them later.